client/src/menu/game/GameScreen.tsx

```diff
@@ -25,6 +26,16 @@ export enum ContentMenu {
     GraveyardMenu = "GraveyardMenu",
     WikiMenu = "WikiMenu",
 }
+
+export const MENU_ELEMENTS = {

```

Comment:
Maybe the enum should just be defined like this. We don't use the values part of the enum